.github/workflows/windows.yml

```diff
@@ -22,8 +22,16 @@ jobs:
           curl https://www.rarewares.org/files/ogg/oggenc2.88-1.3.7-x64.zip --output oggenc.zip &&
           unzip oggenc.zip -d bin/
         shell: bash
+      - name: Configure CMake
+        run: >
+          cmake -Bbuild -DCMAKE_BUILD_TYPE=Release
+          -DCMAKE_TOOLCHAIN_FILE="$env:VCPKG_INSTALLATION_ROOT\scripts\buildsystems\vcpkg.cmake"
+          -DVCPKG_TARGET_TRIPLET=x64-windows-static-md
       - name: Build AV
-        run: msbuild build\VisualStudio\ArrowVortex.vcxproj /p:Configuration=Release /p:Platform=x64         
+        run: cmake --build build --config Release --parallel "$env:NUMBER_OF_PROCESSORS"
+      - name: Collect into a directory
+        if: github.ref_name == 'beta'
+        run: cmake --install build --config Release --prefix ./
```

Comment:
```suggestion
```
If the previous step builds into the `build/` directory it can be referenced in the upload step below without a separate step to collect things